vtkext/private/module/vtkF3DRenderer.h

```diff
@@ -470,12 +471,12 @@ class vtkF3DRenderer : public vtkOpenGLRenderer
 
   vtkSmartPointer<vtkOrientationMarkerWidget> AxisWidget;
 
-  vtkNew<vtkCornerAnnotation> FilenameActor;
   vtkNew<vtkCornerAnnotation> MetaDataActor;
   vtkNew<vtkCornerAnnotation> CheatSheetActor;
   vtkNew<vtkF3DDropZoneActor> DropZoneActor;
   vtkNew<vtkActor> GridActor;
   vtkNew<vtkSkybox> SkyboxActor;
+  vtkNew<vtkF3DUIActor> UIActor;
```

Comment:
is there a single UIActor for all UI elements ?

Reply:
Yes